modules/virt-defining-a-watchdog.adoc

```diff
@@ -9,7 +9,7 @@ Define how the watchdog proceeds when the operating system (OS) no longer respon
 
 .Available actions
 [horizontal]
-`poweroff`:: The virtual machine (VM) powers down immediately.
+`poweroff`:: The virtual machine (VM) powers down immediately. If `spec.running` is in use, or `spec.runStrategy` is not set to `manual`, then the VM will reboot.
```

Comment:
By "in use" do you mean that `spec.running` is `true` or could the attribute be set to either `true` or `false`? That part is not clear.

Reply:
@kbidarkar Please review.

Reply:
Would be good to update "in use" to "true".